drivers/sub8_videoray_m5_thruster/launch/thruster_driver.launch

```diff
@@ -1,4 +1,4 @@
 <launch>
     <node pkg="sub8_videoray_m5_thruster" type="thruster_driver.py" name="thruster_driver"
-        args="--calibration_path=$(find sub8_videoray_m5_thruster)/config/calibration.json" />
+        args="--calibration_path=$(find sub8_videoray_m5_thruster)/config/json_nez_prop.json" />
```

Comment:
Aren't we running with the stock props now? So shouldn't the config be for that?